assets/css/app.css

```diff
@@ -135,6 +139,23 @@ input[type="radio"] {
     line-height: 1.42857143;
     border-radius: 4px;
   }
+
```

Comment:
I assume this code is related to the centering of the modal. Have you tried to do it in the service directly ? See https://github.com/makeusabrew/bootbox/issues/166#issuecomment-104187324